test/cli-validator/mockFiles/thresholds/.zeroWarnings

```diff
@@ -0,0 +1,3 @@
+{
+    "warnings": 0
+}
```

Comment:
Interesting that you made these all hidden files - nothing wrong with that I suppose but the folder will look a little silly when it appears empty